emt4j-common/src/main/java/org/eclipse/emt4j/common/staticanalysis/StaticAnalysisEntry.java

```diff
@@ -0,0 +1,77 @@
+/********************************************************************************
+ * Copyright (c) 2023 Contributors to the Eclipse Foundation
+ *
+ * See the NOTICE file(s) distributed with this work for additional
+ * information regarding copyright ownership.
+ *
+ * This program and the accompanying materials are made available under the
+ * terms of the Apache License, Version 2.0 which is available at
+ * https://www.apache.org/licenses/LICENSE-2.0.
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ ********************************************************************************/
+
+package org.eclipse.emt4j.common.staticanalysis;
+
+import org.eclipse.emt4j.common.JdkMigrationException;
+
+import java.io.File;
+import java.lang.reflect.Constructor;
+import java.net.URL;
+import java.net.URLClassLoader;
+
+public class StaticAnalysisEntry {
+
+    static class MyClassLoader extends URLClassLoader {
```

Comment:
Not a good name

Reply:
Please provide an alternative for reference.

Reply:
SootAnalysisClassLoader